tests/suite/zar/s3/minio.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+mkdir -p data/bucket
+minio server --writeportfile="./port" --quiet --address ":0" ./data &
```

Comment:
I know its not necessary for the ztest usage (since it runs tests in a temp dir), but would you use `mktemp -d` or similar technique to create a unique port file? That would make this script useful if I wanted to run zar on a fake S3 during development.

Reply:
Sure @alfred-landrum. Do I need to worry about delete this tempdir when the test is done?

Reply:
yes, should just be another exit trap